torch/quantization/quantize_fx.py

```diff
@@ -20,13 +20,33 @@ def fuse_fx(graph_module, inplace=False):
     fuser = Fuser()
     return fuser.fuse(graph_module, inplace)
 
-def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic_quant):
+def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic_quant, is_child_module=False):
     _check_is_graph_module(graph_module)
     graph_module = fuse_fx(graph_module, inplace)
     quantizer = Quantizer()
     prepare = quantizer.prepare_dynamic if is_dynamic_quant else quantizer.prepare
-    prepared = prepare(graph_module, qconfig_dict, inplace=True)
-    return prepared
+    return prepare(graph_module, qconfig_dict, inplace=True, is_child_module=is_child_module)
+
+def prepare_child_module_fx(graph_module, qconfig_dict, inplace=False):
```

Comment:
Maybe we can name this `custom_child_module` throughout the code, to clarify the intent?  `child_module` could be confusing as it could mean any module which is a child of an another module.

Reply:
hmm, this can actually work for any child module.

I'm also looking for names for `traceable custom module`, since this overlaps with the previous custom module a bit. I plan to change the name to `accelerator submodule`